gnosis/safe-react
# Broadcast local status updates

## What it solves
Resolves #3223

## How this PR fixes it
When 'local' status updates occur, they are broadcast via a `BroadcastChannel` that then dispatches the payload once more with an extra flag to prevent endless broadcasts.

Note: this is **NOT** supported by Safari.

## How to test it
1. Queue a transaction and ensure that it has sufficient confirmations.
2. Open the same Safe on the queue list in a separate window.
3. Execute the transaction from one window and observe the transaction statuses sync across the both.

## Screenshots
![status sync](https://user-images.githubusercontent.com/20442784/149948131-a78dfa0d-426a-4e62-aa2f-9c1f51ceb596.gif)
